### PR TITLE
Replacing Hige by Lustache

### PR DESCRIPTION
**Pull request by Eynix**
Hi, 

Hige is not correctly parsing templates. Lustache is a port of mustache.js in Lua. It does better than Hige. See the test files attached.
[debug.txt](https://github.com/YunoHost/SSOwat/files/2080147/debug.txt)
[debug_template.txt](https://github.com/YunoHost/SSOwat/files/2080172/debug_template.txt)
[result.txt](https://github.com/YunoHost/SSOwat/files/2080175/result.txt)

Hige has been replaced by Lustache : 
- hige.lua removed
- lustache.lua + lustache/ added
- hige calls and includes replaced 
- "install" file now refers to lustache and the lustache folder

Please note : the "install" has not been tested.

### CODE REVIEW

**Comment by Psycojoker** [comment]
I think we need to credit lustache work, it uses the MIT licence and according to [github short sentence](https://github.com/Olivine-Labs/lustache/blob/master/LICENSE)

> A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.

2018-06-07-133334_222x109_scrot

**Review by Psycojoker — APPROVED** [review]
Need testing but LGTM.

File: init.lua [hunk 1]
```diff
@@ -18,10 +18,10 @@ package.path = package.path .. ";"..script_path.."?.lua"
 local json = require "json"
 local lualdap = require "lualdap"
 local math = require "math"
-local hige = require "hige"
 local lfs = require "lfs"
 local socket = require "socket"
 local config = require "config"
+lustache = require "lustache"
```
Comment (alexAubin) [inline init.lua:24]: Mainly by curiosity, can you comment why there's no "local" here ?

**Review by alexAubin — APPROVED** [review]
Very quick review in diagonal but sounds okay in principle :+1:

**Pull request merged**